app/src/main/res/values-zh/strings.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <string name="app_name">BindingCollectionAdapter</string>
+    <string name="header">我是header</string>
```

Comment:
I understand wanting to have this, but it think everything should be translated or none at all, and I don't think I'd be able to maintain this translation.